deploy/templates/web.template.yml

```diff
@@ -0,0 +1,99 @@
+base_image: "rubybench/rubybench_docker"
+
+update_pups: false
+
+params:
+  home: /var/www/ruby-bench-web
+
+templates:
+  - "templates/redis.template.yml"
+  - "templates/postgres.template.yml"
+  - "templates/web.template.yml"
+
+expose:
+  - "8081:80"
+  - "2228:22"
+
+volumes:
+  - volume:
+     host: /var/docker/shared/ruby-bench-web
+     guest: /shared
+
+hooks:
+  after_redis:
+    - exec:
+        cmd:
+          - useradd rubybench -s /bin/bash -m -U
+
+    - exec:
+        background: true
+        cmd: "sudo -u redis /usr/bin/redis-server /etc/redis/redis.conf --dbfilename test.rdb"
+
+    - exec: "cd /var/www && git pull --rebase origin master https://github.com/ruby-bench/ruby-bench-web.git"
+
+    - exec:
+        cd: $home
+        cmd:
+          - chown -R rubybench $home
+          - mkdir -p /shared/gems
+          - chown -R rubybench /shared/gems
+          - sudo -E -u rubybench bundle install --path=/shared/gems --verbose
+          - sudo -E -u rubybench bundle exec rake
+
+    - file:
+       path: /etc/service/puma/run
+       chmod: "+x"
+       contents: |
+          #!/bin/bash
+          exec 2>&1
+          # redis
+          # postgres
+          cd $home
+          exec sudo -E -u rubybench LD_PRELOAD=/usr/lib/libjemalloc.so.1 bundle exec puma -p 8080 -e production
+
+    - exec: rm /etc/nginx/sites-enabled/default
+
+    - replace:
+        filename: /etc/nginx/nginx.conf
+        from: pid /run/nginx.pid;
+        to: daemon off;
+
+    - file:
+        path: /etc/nginx/conf.d/logster.conf
```

Comment:
should be rubybench.conf , probably should be checked in direct to rubybench web and just copied over to etc/nginx/conf.d instead ...